src/components/header/HeaderComponent.vue

```diff
@@ -217,7 +222,7 @@ const badgeCss = 'fr-badge fr-badge--sm fr-badge--' + props.badgeStyle
               >
                 Fermer
               </button>
-              <HeaderSearch
+              <SearchComponent
                 v-if="searchModalOpened"
                 :search-label="searchLabel"
```

Comment:
Ajouter un `id` pour la recherche mobile.
```suggestion
               id="header-select-search-modal"
               :search-label="searchLabel"
```